### PR TITLE
Update to edger 3.14 and limma 3.28.10

### PR DESCRIPTION
**Pull request by abretaud**
Updating to edger 3.14 as discussed in #798 
@nsoranzo @bgruening I'm not completely sure of myself on this stuff, what do you think about this?

### CODE REVIEW

File: packages/package_bioconductor_edger_3_14_0/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        <install version="1.0">
+            <actions>
+                <action type="setup_r_environment">
+                    <repository name="package_r_3_1_2" owner="iuc">
+                        <package name="R" version="3.1.2" />
+                    </repository>
+                    <package sha256sum="96b3c01e2c6709ee8fb4c43583b22ea0a6f32624b73a9c05d74cac4c443ad342">
+                        https://depot.galaxyproject.org/software/limma/limma_3.28.2_src_all.tar.gz
+                    </package>
+                    <package sha256sum="dc0927def56f2960d49a4550709b184f22e004119298af035ce723a26bae6ed8">
```
Comment (nsoranzo) [inline packages/package_bioconductor_edger_3_14_0/tool_dependencies.xml]: You need to update the sha256sum for 3.14.0, it should be 766a5d1f8e66d28b0c4cc70a72aaa1c5a32047def34d72957ab6fc7cfcf37993

**Comment by nsoranzo** [comment]
:+1:

**Comment by abretaud** [comment]
Hum, should I worry about the failing tests? It looks like it fails early in the galaxy init step...

**Comment by nsoranzo** [comment]
The build failure it's not connected to these changes. We are trying to fix conda testing, see https://github.com/galaxyproject/tools-iuc/pull/798/commits/28ae0edfd5ac135fa86810b02457a546b5afc51f

**Comment by nsoranzo** [comment]
@abretaud If you rebase on latest master and force push, tests should pass now.

**Comment by abretaud** [comment]
Yep, it's green now thanks!

**Comment by abretaud** [comment]
Switching to R 3.2.1 as I think it will help for #821

**Comment by abretaud** [comment]
I updated the limma package too for the same R-version reason

**Comment by nsoranzo** [comment]
@abretaud Some comments:
- What about renaming the repository from `package_r_limma` to `package_bioconductor_limma` and the package from `limma` to `bioconductor-limma`? So it would match the bioconda name.
- The latest version of `bioconductor-limma` in bioconda is 3.28.6, it would be nice to have them in sync (I don't mind if it's 3.28.6 or 3.28.10)
- You need to update name and version of limma dependencies in `tools/trinity/tool_dependencies.xml` and `tools/trinity/run_DE_analysis.xml`

**Comment by abretaud** [comment]
Thank you! It should be better like this
Limma 3.28.10 was not yet merged in bioconda, but it is now: [#1789](https://github.com/bioconda/bioconda-recipes/pull/1789)

**Comment by nsoranzo** [comment]
@abretaud You forgot to push the files in `packages/package_bioconductor_limma_3_28_10/`

**Comment by nsoranzo** [comment]
:+1:

**Comment by abretaud** [comment]
Thank you!

**Comment by abretaud** [comment]
TS are updated now

**Pull request merged**